docs/download.ipynb

```diff
@@ -64,18 +833,58 @@
41YtL0vLyv2wlpfAYdenKXgiCOetB+s3ZCfvbxYlbYgFzRWi6NwGyWrjgj8DJosRI5hy4z6gEuquBiRJRECbyCVPuUl+UtygtPMgG2XRCs7CXwEiNGensPxyYsYdntYfbAii2K8TAaGg8kz83Cr3FykeHDlLmIvkM/XI4r21BFMTbclSfP9GraqvirGrCAfZP6fSzTyc46LCAo4TmwB9wpUTJfL+ChX+nphQVW3JtWxxtqvZrOGNHCwUC7LOa+VqsiG/T0tT85ITSTIRmxLs2NDbeCSzP6/qMHExso+xRB+1IHfyVCtzspMf2fOzvObg7GomPXe+gpGg7vSMUjTirSZdcvDyQapddzMSWsAaQ8YDhJcWVOJ91dePo7ZlpoNb4WkQZlw8nuA3zWFRgolMp5LjoR4jzXsknDVx+OgBCubxcSUinv896MuN3Oa8IqfauJtJ2tw4r6C/TPp5j/PvPuiLCO8vGSEUkqLLwupWYafGcjsOvQ0hCF6rJ9P0PIr2nN7fDGD3+N0NZHjy3lV7EWaoJNvj7WoWd0q4Y7dC6WgBv3Gaao3JaInzVnybntaJe/x9vRE0Z4OdGkz769omTCZXGkgMW4iwYqW8ppEbQ8jTVrafG7lVfxwVcznWB6Ycvz03u7rFzxoHqyQabv4iLMjdpNE15Nrqs7X3k2UEr7Xdy/qsfmoLHAyA+VMj8CMEqJUB8r8mYVmqQm7UkFMRSBSO9y6PF9qMtEOoZeqsCcWoccjhEmA20srmJuZErOFVGvi/Xw8+tJiDYnZQCeLsaED05WWgPFkFsvkO1bh8n+sbDlrIrCESZdzxoOCwJQ7vQGOiYiFWhRIQ6G1XZFUKwaF6Pd6Du85g9xwGCVshwQNWbi8qq6xZ8QlaQxAJFU0wXaetKm7ETnGKY5XdfxoI7cBnHUwbcX4xYOebDjaZQ23N9TzXZiirnCALd/C/U6YA15odL/73ojkNTRLKh25M1kxGApk9ajgxonVa0BQVf58nh9i0+ugUa8rs/QgEXWr2ZkjqDUaCDwPUVedpwKVcuz8GZTsKuIoxKjXhd9VSdGs1wA/QExgD9HWY966slnbf1Xm7dAi+L1SiWl/rPUCrPbYBQAuLlR35VpzrjnPYUCuc6kknG35TilZapICmT5UXXmDIbyBJ/iAycWZRwr4fxESb5glsPUYYdCTz+O6bey3xi2u1z7dw5IuonSIjJuTpCTz+CQKBLQbcdbuToigF+0Z37/f3zGdODGlkrlsjhNlslA8NzsDL062FZgvfy0medKYmo72ZHxpdhuC8KHEaznkph9cDX+qxFsONx4S5paIR7A1H3OTR5FmnIP0hbSuRQO4bAuFG3k7RdkHqvf15G/waQRPPaX//iwGW7TnJkM5GLynqYA1TmWi7vNh1KDnZwPM1YZyARKkxaqY4K397Wny2OhRXBLtaQ0bQxPxvjlvAdIqKtrxYGt7oaa2NzSd2PSMR4C2yGfV8eG6I0fx3HM1BQgr5DqnyornyWA70bHL2Oyto16ZFCWr/c9dK03KfDMzPWhjn60IU+wTWa0Y6PcH+Af/4B9ge2sbtXoNz5w+jV/79V+D41Zh2S50Hm0HdjXGeb2ERovcdRNGqQGEHn73H//f8Xf/7t9Bq7UAI7Xxi5uqzfz8XBWNyTYsP8ICOcQW54EpdJCPaEvFuzYc4O21DWVDJou0Qr6OJ11+Mu7IuVDzZicF6STdkNJdnIRlV6VdTblMVmCk9T0ufBkTZbAyHV1P3c+/era9I6vHGXLkm8Jn1rmJ5iLJN0PBD8PF5ZWtHc1lvuYHq+p5z07p+MkdNYJ65egEnJKBd3JjhDPTLmq5PvTudzEm3K8Z8FIuzB8/eXBdXRv2oaUDmHDQ722hXplG1amhbB9efUSpj2DUw3BEcNnuupYMhlh+7wpWm00Ycy2cmJpC+TFr1UF82f1BjjXj+KSLHimY0k7PxBVHGNIahSTMPabzEgJS3cvGYJFDH2C3ToUvgu0+mTn9ZxVKatJEvdRCwM2MpkvleVjwPkgzNWpR7PF8c5b/nsk7pOJX4gjY7cOc/vbCfHUn6ZKjzi4SE+se8OW+r4ot+4/TaucGh+pi+/OX/TEtBJ/4Gzs1d14G2QHFA/KKl7tPIvp4IdSr1EZNsNVZlTagWuB4Y6gd6s7OIxdtfxix+cuNRMtgCIUzt6QZC1L2Q90Bl8dSSgm3L2aS5sLfKjfh2n3MVn185chuVayQ06oiZlLe7ydcJEdBUM/4ktB7Aavi3fY0K+PxoK50TdDTBL4YexL1znvSmDRU1cujCM5jjtUjAYeNIk2kMpmQFWjr4KqY4iJLY7aLO05SEzSw0NCqjzDVnMV2fxULUyel/TweLvEGbNtqvtBOouEQlutCF+ehva9Llal+r4/f/PXflPnUb//2P8L5i+dRq1Tx05++Luo8X/nqVzA7O4erP/s5XnzxBfR6A3x4+TKefe45DAeUJ3SwsdHDj3/4Y7x9b0EWCarmLK+s4kc//CEsy8Z3f+1X4fshHiwvY2VpCYunTyNst7E08mSeyyDVQiXgXWWhU/U65iqKkxmKyYKJWbeEeHs38ZbdSVhWDV1/C5lmwKdQvL5LLSJ/ks89nhjo4OPaJn58a/d5vnl6V52qCIsOY3GsFL8oVOQ4aJQn8MEylbmALx+fgVsy8P6KUgtq5uvOsUqGozMnxC/3rZ9d37Gh3Cv7yGtZUUG4lvQTTboO49Wu4sWqhPhQMtp37XixUjWiNOzIp2NNH6fnn4PrKErV/uCst9O/B9/chpZSnGLM/zZNEfkBsLKKqDdAv1yF+ylAQ0UUjlUvL9bFam/D92VmPk8Od+GGw00PRYfyNYpa5fwNk8h4xGGEjXsrqDRraEw2pYvzhQ/KProtOR4XxDKs9OrI4g0YSQla/v2wl6Vn+YY7ZTFFneYUH+TmHC/kSHxeNyNKdprGTvv6oGBnohNECjvxhCE5a3/2/gTxxIn32OwzOSpydwBN0jGFNLhsU
```

Comment:
Would it be worth having a follow-up suggestion that you can even browse the item's assets?

```suggestion
    "print(item.get_assets())"
```